models/public/erfnet/accuracy-check.yml

```diff
@@ -0,0 +1,49 @@
+models:
+  - name: ERFNet
+    launchers:
+      - framework: onnx_runtime
+        adapter:
+          type: segmentation
+          make_argmax: True
+    datasets:
+      - name: erfnet_data
+        preprocessing:
+          - type: resize
```

Comment:
are you tested your configuration with onnx_runtime launcher?

if info about channels order and scale in your readme correct, then it should give wrong result because preprocessing lead that as images provided to model input as BGR in [0, 255] range, but according your docs should be RGB and [0, 1] range

Reply:
channel order should is "BGR". So we don't need --reverse_input_channels.I have modified it in the README